osvr/RenderKit/RenderManagerOpenGL.cpp

```diff
@@ -23,15 +23,18 @@ Sensics, Inc.
 // limitations under the License.
 
 #include "RenderManagerOpenGLVersion.h"
+#ifndef OSVR_ANDROID
 #include <SDL.h>
```

Comment:
It would be good to have a capability-based check here, like OSVR_SDL, rather than an OS-specific one.  This will let us compile on Windows without SDL, for example, using the Qt example program.  Same #define for the other SDL guards below.